realsense2_camera/launch/includes/nodelet.launch.xml

```diff
@@ -67,18 +67,6 @@
   <arg name="aligned_depth_to_infra2_frame_id" default="$(arg namespace)/camera_aligned_depth_to_infra2_frame"/>
   <arg name="aligned_depth_to_fisheye_frame_id" default="$(arg namespace)/camera_aligned_depth_to_fisheye_frame"/>
 
-  <rosparam param="/diagnostic_aggregator/analyzers" subst_value="true">
-    realsense2_camera_manager:
-      type: diagnostic_aggregator/GenericAnalyzer
-      path: realsense2_camera_manager
-      contains: realsense2_camera_manager
-      discard_stale: true
-      timeout: 3
-      expected:
-        - "$(arg namespace)/$(arg manager)_depth: Frequency Status"
-        - "$(arg namespace)/$(arg manager)_infra1: Frequency Status"
-        - "$(arg namespace)/$(arg manager)_infra2: Frequency Status"
```

Comment:
is `depth` the only one thats failing? lets keep all the others if thats the case

Reply:
Okay. And there's no `infra2`, so I removed it.